Make.inc

```diff
@@ -167,6 +167,12 @@ else
 JULIA_COMMIT := $(JULIA_VERSION)
 endif
 
+# Override `JULIA_COMMIT` to `JULIA_VERSION` if we're on a tagged commit
+ifeq ($(shell git describe --tags --exact-match > /dev/null 2>&1 && echo true),t
```

Comment:
Accidental linebreak? :P

Reply:
I tested it on release-0.6, and then copy-pasted it for master, which explains it.